api/src/main/java/net/md_5/bungee/api/plugin/PluginManager.java

```diff
@@ -295,12 +295,11 @@ private boolean enablePlugin(Map<PluginDescription, Boolean> pluginStatuses, Sta
         // do actual loading
         if ( status )
         {
-            try
-            {
-                URLClassLoader loader = new PluginClassloader( new URL[]
```

Comment:
It's intended that the PluginClassloader is in use for the whole lifetime of the plugin.